Nomic_Analysis/1.Exploratory_Analysis/PBMC/nbconverted/analysis_PBMC.py

```diff
@@ -0,0 +1,248 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import pathlib
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import seaborn as sns
+from matplotlib.backends.backend_pdf import PdfPages
+from sklearn import preprocessing
+
+# In[2]:
+
+
+path = pathlib.Path("../../Data/clean/Plate2/nELISA_plate_430420_PBMC.csv")
```

Comment:
Consider renaming `path` to clarify what this path is for.

Reply:
renamed path